src/mozmlops/templates/template_flow.py

```diff
@@ -0,0 +1,111 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from mozmlops.artifact_store import ArtifactStore
```

Comment:
This won't work unless `mozmlops` is installed: I'd add a one line comment above this line to mention it. It shouldn't be a big deal, this is going to change once the package gets published and I doubt anyone is going to use this in the next week or so.

Reply:
Can do!